### ISSUE TITLE
[QUESTION]: dellemc.openmanage.ome_firmware not working with baseline attribute

### ISSUE THREAD

**Issue by alertGIT** [dell/dellemc-openmanage-ansible-modules#539]
### How can the team help?
I have some trouble with the module **dellemc.openmanage.ome_firmware** to update firmware of servers when pointing to a baseline.
If providing the dup package directly it is working.
Maybe you have an idea why Update Job on OME will not be triggered.
The playbook execution seems to be hanging . No job is seen in OME.
User access to OME is using the “admin” user.

**Details: ?**
ansible [core 2.15.3]
python version = 3.9.16 (main, May 31 2023, 12:21:58) [GCC 8.5.0 20210514 (Red Hat 8.5.0-18)] (/bin/python3)
OME Version: Version 3.10.2 (Build 13)
iDRAC Version: 6.10.80.00

**Ansible Playbook**

```
---
- hosts: "{{ hosts }}"
  gather_facts: False

  tasks:
  - name: Stage firmware for the next reboot using baseline name
    dellemc.openmanage.ome_firmware:
      hostname: "{{ ome_hostname }}"
      username: "{{ ome_username }}"
      password: "{{ ome_password }}"
      validate_certs: False
      baseline_name: ansible_server_baseline
      schedule: StageForNextReboot
    register: result
    delegate_to: localhost
```
**Output**
```
TASK [Stage firmware for the next reboot using baseline name] ************************************************************************************************************************************
task path: /home/ansible/ansible-projects/workshop/ome_firmwarebaseline2.yml:6
<localhost> ESTABLISH LOCAL CONNECTION FOR USER: ansible
<localhost> EXEC /bin/sh -c 'echo ~ansible && sleep 0'
<localhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/ansible/.ansible/tmp `"&& mkdir "` echo /home/ansible/.ansible/tmp/ansible-tmp-1692699182.978376-4166748-261897700904951 `" && echo ansible-tmp-1692699182.978376-4166748-261897700904951="` echo /home/ansible/.ansible/tmp/ansible-tmp-1692699182.978376-4166748-261897700904951 `" ) && sleep 0'
Including module_utils file ansible/__init__.py
Including module_utils file ansible/module_utils/__init__.py
Including module_utils file ansible/module_utils/basic.py
Including module_utils file ansible/module_utils/_text.py
Including module_utils file ansible/module_utils/common/_json_compat.py
Including module_utils file ansible/module_utils/common/__init__.py
Including module_utils file ansible/module_utils/common/_utils.py
Including module_utils file ansible/module_utils/common/arg_spec.py
Including module_utils file ansible/module_utils/common/file.py
Including module_utils file ansible/module_utils/common/locale.py
Including module_utils file ansible/module_utils/common/parameters.py
Including module_utils file ansible/module_utils/common/collections.py
Including module_utils file ansible/module_utils/common/process.py
Including module_utils file ansible/module_utils/common/sys_info.py
Including module_utils file ansible/module_utils/common/text/converters.py
Including module_utils file ansible/module_utils/common/text/__init__.py
Including module_utils file ansible/module_utils/common/text/formatters.py
Including module_utils file ansible/module_utils/common/validation.py
Including module_utils file ansible/module_utils/common/warnings.py
Including module_utils file ansible/module_utils/compat/selectors.py
Including module_utils file ansible/module_utils/compat/__init__.py
Including module_utils file ansible/module_utils/compat/_selectors2.py
Including module_utils file ansible/module_utils/compat/selinux.py
Including module_utils file ansible/module_utils/distro/__init__.py
Including module_utils file ansible/module_utils/distro/_distro.py
Including module_utils file ansible/module_utils/errors.py
Including module_utils file ansible/module_utils/parsing/convert_bool.py
Including module_utils file ansible/module_utils/parsing/__init__.py
Including module_utils file ansible/module_utils/pycompat24.py
Including module_utils file ansible/module_utils/six/__init__.py
Including module_utils file ansible/module_utils/urls.py
Including module_utils file ansible/module_utils/compat/typing.py
Including module_utils file ansible_collections/dellemc/openmanage/plugins/module_utils/ome.py
Including module_utils file ansible_collections/__init__.py
Including module_utils file ansible_collections/dellemc/__init__.py
Including module_utils file ansible_collections/dellemc/openmanage/__init__.py
Including module_utils file ansible_collections/dellemc/openmanage/plugins/__init__.py
Including module_utils file ansible_collections/dellemc/openmanage/plugins/module_utils/__init__.py
Using module file /home/ansible/.ansible/collections/ansible_collections/dellemc/openmanage/plugins/modules/ome_firmware.py
<localhost> PUT /home/ansible/.ansible/tmp/ansible-local-4166727i8lnkto4/tmpnszsuovy TO /home/ansible/.ansible/tmp/ansible-tmp-1692699182.978376-4166748-261897700904951/AnsiballZ_ome_firmware.py
<localhost> EXEC /bin/sh -c 'chmod u+x /home/ansible/.ansible/tmp/ansible-tmp-1692699182.978376-4166748-261897700904951/ /home/ansible/.ansible/tmp/ansible-tmp-1692699182.978376-4166748-261897700904951/AnsiballZ_ome_firmware.py && sleep 0'
<localhost> EXEC /bin/sh -c '/bin/python3 /home/ansible/.ansible/tmp/ansible-tmp-1692699182.978376-4166748-261897700904951/AnsiballZ_ome_firmware.py && sleep 0'
```

**Comment by alertGIT** [comment]
Bug already reported.
#494

**Comment by sachin-apa** [comment]
Closing this as #494 is Closed and OME 4.0 is released. Please feel free to reopen if any.
